### PR TITLE
Task1

### PR DESCRIPTION
**Pull request by Antontelesh**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+import './App.css';
+
+function App() {
+  return (
+    <div className="container">
+      <div className="wrapper">
+      <FunctionComponent></FunctionComponent>
+      <ReactComponent></ReactComponent>
+      <CreateElement></CreateElement>
+      <PureComponentContainer></PureComponentContainer>
```
Comment (Antontelesh) [inline src/App.js]: FYI, if your component does not take child elements it's possible to have a shorthand syntax:

```jsx
<PureComponentContainer />
```

File: src/FunctionalComponent/FunctionalComponent.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+import React from 'react'
+import './FunctionalComponent.css'
+
+const functionalComponent = () => {
```
Comment (Antontelesh) [inline src/FunctionalComponent/FunctionalComponent.js]: it's better to name components from a capital letter
```suggestion
const FunctionalComponent = () => {
```

**Review by Antontelesh — COMMENTED** [review]
Everything is okay, just a couple of minor comments

File: src/App.js (hunk 1 above)
Comment (KVManeeshReddy) [reply src/App.js]: used short hand syntax

File: src/FunctionalComponent/FunctionalComponent.js (hunk 2 above)
Comment (KVManeeshReddy) [reply src/FunctionalComponent/FunctionalComponent.js]: Used Capital letter for components name.

**Pull request merged**